src/iceberg/request-upgrader.service.js

```diff
@@ -108,7 +109,7 @@ export default class IcebergRequestUpgrader {
         data,
         headers,
         status,
-      ) => ({ data, headers: headers(), status }),
+      ) => ({ data: angular.fromJson(data), headers: headers(), status }),
```

Comment:
```suggestion
      ) => ({ data: JSON.parse(data), headers: headers(), status }),
```
?

Reply:
What if `data` is not parsable JSON ?

Reply:
@marie-j @FredericEspiau 

It's the default behaviour when transformResponse is not override:
https://docs.angularjs.org/api/ngResource/service/$resource

So let use `angular.fromJson` ;)